models/spring-ai-ollama/src/main/java/org/springframework/ai/ollama/management/PullModelStrategy.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.ai.ollama.management;
+
+/**
+ * Strategy for pulling Ollama models.
+ *
+ * @author Thomas Vitale
+ * @since 1.0.0
+ */
+public enum PullModelStrategy {
+
+	/**
```

Comment:
When pulling models, there are two options here: always (latest model version) and when_missing (the model could be stale).